app/views/components/docs/important-metadata.yml

```diff
@@ -0,0 +1,17 @@
+name: Important metadata
+description: List of document-type specific metadata
+body: |
+    A replacement for the [metadata component](https://govuk-static.herokuapp.com/component-guide/metadata) with only the format specific details. This component separates facet metadata from [publisher metadata](/component-guide/publisher-metadata).
+
+    Part of the universal navigation design.
+
```

Comment:
I'm wondering if we need another accessibility criteria about the contrast of the text against the background, since it's white on blue, which isn't common on the site.

Reply:
Added here: https://github.com/alphagov/government-frontend/pull/541/commits/5c5d697ff5d7773e97554dd9d7e149e070c090fd